### DISCUSSION TITLE
Open iframe inside a specific div

### DISCUSSION THREAD

**Discussion by ThaisCursi** [fancyapps/ui#172] [General]
Good afternoon, is it possible for me to choose inside which class the fancybox should open?
I have some places that open fancybox and I would like it to open it inside some specific class so that I can define the div measure in css.

I'm upgrading to version 4.0

**Comment by fancyapps** [comment]
Hi, Currently you can set only a custom ID for each slide and then use it for the styling. See second example here - https://codesandbox.io/s/jolly-shamir-p5kdu?file=/index.html

HTML 
```html
<a
  href="./src/iframe.html"
  data-fancybox
  data-type="iframe"
  data-id="iframeB"
  data-auto-size="false"
>
  Example
</a>
```
CSS
```css
#iframeB {
  max-width: 480px;
  max-height: 320px;
}
```

If you really want to have a class name instead of the id, I can add support. But I don't think it really matters, because you'll (probably) only have one element on the page with that ID at a time, so you can reuse it for multiple links.

**Comment by ThaisCursi** [comment]
Hi Janis Skarnelis.

Thank you for answering, it helped me a lot.
